test/snapshots/tools/generate-inline-test.mjs

```diff
@@ -15,4 +15,8 @@ const template = resolve(dir, './inline-test-template.js');
 
 (async () => {
   await generateInlineTest(template, filepath)
+  await generateInlineTest(
+    resolve(dir, './inline-test-template-concurrent.js'),
+    resolve(dir, '../test-update/inline-test-template-concurrent.test.js'),
+  )
```

Comment:
This test case is mostly taken from https://github.com/vitest-dev/vitest/pull/4783. Thanks @martypdx!